src/physics/euler.cpp

```diff
@@ -296,12 +287,7 @@ template <int dim, int nstate, typename real>
 inline real Euler<dim,nstate,real>
 ::compute_entropy_measure ( const real density, const real pressure ) const
 {
-    if (density < 0.0) {
```

Comment:
Is it always the case that `compute_entropy_measure ( const real density, const real pressure ) ` is called through `compute_entropy_measure ( const std::array<real,nstate> &conservative_soln )` and not directly?

Reply:
I'm not sure that we always use the `compute_entropy_measure ( const std::array<real,nstate> &conservative_soln )` form. It does make more sense to move the check to `compute_entropy_measure ( const real density, const real pressure )`.

This also falls under @jbrillon 's comment about changing the behaviour of the entropy check, which I'm still thinking about. I will fix this after addressing that comment.

Reply:
looks good to me